[prim_util] Do not use $clog2() in Xcelium

imphil opened the pull request:
Xcelium seems to have trouble with the use of $clog2() in
`prim_util_pkg::vbits()`. Fortunately, we can provide a drop-in
replacement which keeps the interface identical.

This approach allows us to keep the interface/function signature
identical, with the option to remove the workaround once we understand
the Xcelium issue better.

If we switched to a macro we had to change all call sites, and make
`prim_util` an include file again.

Fixes #2597

imphil commented:
@rasmus-madsen once you can reproduce the issue, can you check if this pull request solves the problem?

rasmus-madsen commented:
the fix is good - but we need to define XCELIUM somewhere when running.
I suggest that we set this in the xcelium.hjson file until we all have access to xcelium 19.10.001

imphil commented:
Here we go. I checked the specs, and our use of $clog2() is valid since the SV2005, which is also the version where $clog2() first appeared (Section 12.3.3 Constant function calls). Seems like Xcelium has simply missed that spec change.

I added the XCELIUM define as proposed by @rasmus-madsen and updated the comment inside the file. 

@rasmus-madsen Can you give this change a last try to see if now everything works as expected?

imphil commented:
@rasmus-madsen confirmed that the UART regression is working in Xcelium with this fix in #2597. Thanks everyone for testing and tracking this issue down, I think we have a good solution for now until we can require Xcelium 19.10.
